### ISSUE TITLE
planet dimensions wrong id

### ISSUE THREAD

**Issue by VastayanWings** [BlesseNtumble/GalaxySpace#372]
**Do not remove this template, without it your issue will not be considered!**

**Please fill in the form below:**
------------------------------------------------------------------------
1. Minecraft version: 12.2.2
2. Galacticraft version: 4.0.2.237
3. GalaxySpace version: 2.0.10
4. AsmodeusCore version (for 2.0.1 version and above): 0.0.11
5. Side (Single player (SSP), Multiplayer (SMP), or SSP opened to LAN (LAN)):SSP

------------------------------------------------------------------------
Description of the issue:

everytime i try to fly to mars with a tier 2 rocket i end up getting send to venus instead
i didnt tried any other planet/moon 
config for galacticraft and galaxy space are both default/unedited
------------------------------------------------------------------------
Screenshot/Video:

------------------------------------------------------------------------
Attached log file (or url on pastebin.com):

[latest.log](https://github.com/BlesseNtumble/GalaxySpace/files/3704425/latest.log)

------------------------------------------------------------------------

**Comment by BlesseNtumble** [comment]
disable Venus space station in dimensions.conf

**Comment by Polosolo12** [comment]
I have also had this issue except instead it seems the entire map galaxy map gui does not work.


At first I was under the impression that I was choosing planets that were still a WIP (the ones out of the Sol system.) In creative, I launched to Proxima B, only to have the game crash. However, when I relaunched, I was spawned on Callisto, which I only knew by opening the debug screen and looking at the item IDs. Next I tried Barnada C, which caused the same result, except when re-launching I spawned into a very glitched dimension that was seemingly broken. This time I used the /forge tps command to see what dimension I was on, and it showed the dimension ID for mercury. Curious, I teleported to that dimension ID using Mystcraft /tpx command and I just respawned in the same broken dimension. 

At this point, I removed all my mods except for Galacticraft and Galaxy Space to make sure it was not a compatibility issue. I launched to Mercury, and my game once again crashed. This time when I reopened it I respawned on Callisto B. Using forge /tps it said once again I was on a different planet than what I chose. As a final test I launched to Mars, and my game just froze on "taking one giant leap."

log for first crash:
[crash-2019-11-04_20.35.11-server.txt](https://github.com/BlesseNtumble/GalaxySpace/files/3807469/crash-2019-11-04_20.35.11-server.txt)

log for second crash:

[crash-2019-11-04_20.56.33-server.txt](https://github.com/BlesseNtumble/GalaxySpace/files/3807473/crash-2019-11-04_20.56.33-server.txt)
